package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@deskpro/app-sdk",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Don't need to bump this version number as the build/publish process will do this. Please revert